src/30_Advanced/Star_Rating.html

```diff
@@ -47,7 +47,6 @@
       -webkit-tap-highlight-color: rgba(0,0,0,0);
       -webkit-tap-highlight-color: transparent;
       /* border: 1px solid red;  /* debug */
-      margin-bottom: 1em;
     }
 
     /* the stars */
```

Comment:
ok, we could improve a little bit by:
- adding proper margins
- change "Things to try" to something like "instructions to interact with the start rating widget / to experiment the star rating widget"